email/spool.rst

```diff
@@ -141,3 +141,16 @@ You can also set the time limit in seconds:
 Of course you will not want to run this manually in reality. Instead, the
 console command should be triggered by a cron job or scheduled task and run
 at a regular interval.
+
+.. caution::
+
+    When you use the filesystem for spooling, messages are serialized with
```

Comment:
The explanation is very clear to me ... but maybe we'd need at the beginning a short phrase explaining the `Swift_Message_<someRandomCharacters>` thing that you mention later. Otherwise, this may be confusing:

```
messages are serialized with the message class name.
```

Which class name? I don't create classes to send messages! You don't create them ... but SwiftMailer does that for you.